tests/conftest.py

```diff
@@ -281,3 +284,84 @@ def pytest_runtest_protocol(item):
             ihook.pytest_runtest_logfinish(nodeid=nodeid, location=item.location)
 
         return True
+
+
+# source code fixtures
+
+
+def _run(cmd):
+    return subprocess.check_output(cmd, shell=True)
+
+
+@contextlib.contextmanager
+def create_package(directory, pyproject, setup):
```

Comment:
Does this need to be in the main conftest for all other tests?

Since these seem very test specific, we should move these to be co-located with the tests either in `tests/tracer/conftest.py` or `tests/tracer/test_gitmetadata.py`.

We don't want these fixtures to potentially conflict with others (`temp_dir` or `example` seem like common enough names).

Reply:
Yes, it used in `tests/sourcecode/` and in `tests/tracer/`

Reply:
oh, I think I missed that part. I saw the deletion and assumed it was going away.